tests/checkpoint/test_checkpoint.py

```diff
@@ -4431,7 +4426,7 @@ def test_newstyle_checkpoint_instantiates_and_produces_a_correct_validation_resu
     checkpoint = context.get_checkpoint(name="my_checkpoint")
 
     results = checkpoint.run()
-    assert results["success"] == False
+    assert not results["success"]
```

Comment:
If we're explicitly checking for a bool, can we use `is False`?

Reply:
@cdkini We can; I am not sure if it is necessary.  Please let me know what you think (happy to make it as you describe).  Thanks.